src/Domain/Person.php

```diff
@@ -239,11 +244,15 @@ class Person
      * Historically we didn't require this, but will for new accounts in future and may create
      * an optional verification process that holders of old accounts can use.
      */
-    #[ORM\Column]
-    public bool $email_address_verified = false;
+    #[ORM\Column(nullable: true, type: 'datetime_immutable')]
+    public ?\DateTimeImmutable $email_address_verified = null;
```

Comment:
when there's a choice I think it's very often better to save a timestamp in the db instead of a bool - very little extra complexity and likely to be useful to do manual queries to see when someone verified their email address.